model/websocket_request_test.go

```diff
@@ -6,20 +6,18 @@ package model
 import (
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestWebSocketRequest(t *testing.T) {
 	m := WebSocketRequest{Seq: 1, Action: "test"}
 	json := m.ToJson()
 	result := WebSocketRequestFromJson(strings.NewReader(json))
 
-	if result == nil {
-		t.Fatal("should not be nil")
-	}
+	require.NotNil(t, result, "should not be nil")
```

Comment:
nit: I think we can safely remove the comments here, as the `NotNil` check would make the expectations clear in case of error.

```suggestion
	require.NotNil(t, result)
```